srcCpp/customType.h

```diff
@@ -0,0 +1,32 @@
+#ifndef __CUSTOMTYPE_H__
+#define __CUSTOMTYPE_H__
+/*
+ * (c) 2005-2017  Copyright, Real-Time Innovations, Inc. All rights reserved.
+ * Subject to Eclipse Public License v1.0; see LICENSE.md for details.
+ */
+#define Q(x) #x
+#define QUOTE(x) Q(x) // Used to set quotation marks
+
+#include QUOTE(RTI_CUSTOM_TYPE_FILE_NAME_SUPPORT)
+
+#include "perftest.h"
+#include <string>
+#include <new>          // std::bad_alloc
+
```

Comment:
You might want to add an explanatory text after the includes explaining what is this file for and how to use it and maybe a reference to the documentation.

Reply:
Also for the .cxx